slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -38,12 +38,26 @@ using viam::service::slam::v1::SLAMService;
 namespace viam {
 
 static const int checkForShutdownIntervalMicroseconds = 1e5;
+
+// rgb value representing no input from a Cario pained map
+static const unsigned char defaultCairosEmptyPaintedSlice = 102;
+static const int jpegQuality = 50;
+// byte limit on GRPC, used to help determine sampling skip_count
+static const float maximumGRPCByteLimit = 32 * 1024 * 1024;
```

Comment:
why is this a float?

Reply:
wanted a float for the division in the sampling math, but i should just add that to the actual math and not the variable declaration here